src/styles/blocks/card.css

```
@@ -0,0 +1,93 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--white-color);
+
+  --white-color: #fff;
+  --aqua-color: #00acdc;
+  --dark-blue-color: #060b35;
+  --light-grey-color: #616070;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--dark-blue-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--light-grey-color);
+}
+
+.card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  padding-top: 2px;
+  line-height: 14px;
+  font-size: 10px;
+  color: var(--dark-blue-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__name {
+  line-height: 18px;
+  color: var(--light-grey-color);
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark-blue-color);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--white-color);
+  text-decoration: none;
+
+  border: 1px solid rgba(255, 255, 255, 0);
+  border-radius: 5px;
+  background: var(--aqua-color);
+
+  transition: 0.3s ease;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```